evil-owl.el

```diff
@@ -86,7 +86,7 @@ Possible format specifiers are:
 - %s: the register's contents"
   :type 'string)
 
-(defcustom evil-owl-local-mark-format " %m: [l: %-5l, c: %-5c]"
+(defcustom evil-owl-local-mark-format " %m: %s"
```

Comment:
As I said in my reply, I think we should keep the line/column numbers while adding a line context option. As a result, I'd prefer if the default formats stayed the same.